staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -92,17 +92,21 @@ func (s *EtcdOptions) Validate() []error {
 
 	for _, override := range s.EtcdServersOverrides {
 		tokens := strings.Split(override, "#")
-		if len(tokens) != 2 {
-			allErrors = append(allErrors, fmt.Errorf("--etcd-servers-overrides invalid, must be of format: group/resource#servers, where servers are URLs, semicolon separated"))
+		if len(tokens) != 3 {
```

Comment:
this breaks existing apiserver invocations, which is not acceptable. this needs to honor existing invocations and result in identical behavior on upgrade